kata_agent.go

```diff
@@ -335,6 +410,45 @@ func (k *kataAgent) startPod(pod Pod) error {
 		hostname = hostname[:maxHostnameLen]
 	}
 
+	//
+	// Setup network interfaces and routes
+	//
+	networkNS, err := pod.storage.fetchPodNetwork(pod.id)
+	if err != nil {
+		return err
+	}
+	interfaces, routes, err := k.generateKataInterfacesAndRoutes(networkNS)
+	if err != nil {
+		return err
+	}
```

Comment:
Please add a blank line to let the code breathe :smile:
And globally, this comment applies to every place in the code where there are not enough blank lines (for readability)

Reply:
Will add some air.